tests/bin/rpminspect.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+
+# Don't run metadata check as we can't know the build host subdomain
+# of CI runners in advance to add to an allow list
+for f in build/RPMS/*; do
+  echo "Running RPMLint on $f"
```

Comment:
Could we use something like this?
```
echo "::group::Running RPMInspect on $f"
...
echo "::endgroup::"
```
That way it would be easier to find the logs for a particular package. See:
https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#grouping-log-lines

Reply:
Noice, I was unaware of this syntax. Good [nerd sniping](https://xkcd.com/356/), I'll be checking this out.